chart/openfaas/values.yaml

```diff
@@ -1,8 +1,12 @@
 functionNamespace:
 async: false
 armhf: false
+
+# Service
```

Comment:
Why are we deleting the comments?

Reply:
I did not delete comments, I added more of them